packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.styles.ts

```diff
@@ -105,3 +108,15 @@ export const SwitchNftWrapper = styled.div`
   justify-content: start;
   margin-bottom: ${sizes(4)};
 `
+
+export const VideoLink = styled(Link)`
+  font: ${cVar('typographyDesktopT200')};
+  letter-spacing: ${cVar('typographyDesktopT200LetterSpacing')};
+  text-transform: ${cVar('typographyDesktopT200TextTransform')};
+  color: ${cVar('colorCoreBlue300')};
+  text-decoration: none;
+`
+
+export const DescriptionTextArea = styled(TextArea)`
+  margin-top: ${({ disabled }) => disabled && '40px'};
```

Comment:
let's use `sizes` here. Also if we leave that, we will have `margin-top: false`, which is not great.